content/docs/connectors/keystone.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Authentication Through OpenStack Keystone"
+linkTitle: "OpenStack Keystone"
+description: ""
+date: 2020-09-30
```

Comment:
This date seems incorrect -- v2.14.0 was released on Jan 11, 2019, but for documentation we may want to use the day the page was published or written.

Reply:
Overlooked it, thank you. Need time to get used to it.